templates/webapi/test/result.html.ep

```diff
@@ -16,7 +15,9 @@
 % content_for 'ready_function' => begin
     $('.timeago').timeago();
     setupResultButtons();
-    setupResult(<%= $job->id %>, '<%= $job->state %>', '<%= $job->result %>', '<%= url_for('status', testid => $job->id) %>', '<%= url_for('test_details', testid => $job->id) %>');
+    setupResult(<%= $job->id %>, '<%= $job->state %>', '<%= $job->result %>',
+    '<%= url_for('status', testid => $job->id) %>',
+'<%= url_for('test_details', testid => $job->id) %>');
```

Comment:
looks like this is also just whitespace change and should be reverted unless I overlook the actual functional change

Reply:
reverted. however i think this is produced by the tidy-js. **this change is ready for review**